library/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.firebase.ui">
 
+    <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
I've so far refrained from requiring this in the library, to make it explicitly the task of the app builder to require INTERNET access. Do you feel strongly on having this be implied?